### DISCUSSION TITLE
列表页里面怎么显示关联关系的数据

### DISCUSSION THREAD

**Discussion by krvipsoft** [slowlyo/owl-admin#119] [Anything]
amis()->SelectControl('country_id', '国家')
				->options(CountryService::make()->query()->get(['id', 'name']))
				->labelField('name')
				->valueField('id')
				->required(),
				这样在form里正常显示没有问题
如何在list和detail里显示
amis()->TableColumn('country_id', '国家'),
amis()->TextControl('country_id', '国家')->static(),

**Comment by slowlyo** [comment]
- 通过模型获取器添加显示字段
- 可以通过模型管理, 显示多层级的数据, 比如: user.name

**Comment by krvipsoft** [comment]
有具体的介绍代码吗？

**Reply by slowlyo** [reply]
可以看[demo站点](https://demo.owladmin.com/admin)中的 <开发示例>

**Comment by krvipsoft** [comment]
嗯，已解决
